Rubberduck.Core/UI/Controls/PersistGroupExpandedStateBehavior.cs

```diff
@@ -47,10 +47,9 @@ protected override void OnAttached()
             var states = GetExpandedStateStore();
             var expanded = !states.ContainsKey(GroupName ?? string.Empty) ? (bool?)null : states[GroupName ?? string.Empty];
 
-            if (expanded != null)
-            {
-                AssociatedObject.IsExpanded = InitialExpandedState;
-            }
+            AssociatedObject.IsExpanded = expanded != null
+                ? expanded.Value
+                : InitialExpandedState;
```

Comment:
Assignment can be simplified
`AssociatedObject.IsExpanded = expanded?.Value ?? InitialExpandedState;`

Reply:
VS erroneously says `expanded?.Value` is an error [CS1061](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1061).

Reply:
Sorry, my mistake...`expanded` is a `bool?`.  Should be:
`AssociatedObject.IsExpanded = expanded ?? InitialExpandedState;`